pig/src/main/java/org/apache/iceberg/pig/IcebergPigInputFormat.java

```diff
@@ -245,8 +246,7 @@ private boolean advance() throws IOException {
 
     private Object convertPartitionValue(Type type, Object value) {
       if (type.typeId() == Types.BinaryType.get().typeId()) {
-        ByteBuffer buffer = (ByteBuffer) value;
-        return new DataByteArray(buffer.get(new byte[buffer.remaining()]).array());
+        return new DataByteArray(ByteBuffers.toByteArray(((ByteBuffer) value));
```

Comment:
Looks good.

Reply:
Fixed a linter error. Wasn't aware we had any breaking linting in the repo. Would love to help enable more of that!